File: 58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        length = len(s)
        i = length - 1
        count = 0
        
        while(s[i] == ' ' and count == 0):
            i -= 1

        while(i >= 0 and s[i] != ' '):
            count += 1
            i -= 1

            if(i < 0):
                return count
        
        return count

File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int length = s.size();
        int i = length-1;
        int count = 0;
        while(s[i] == ' ' && count == 0){
            i--;
        }
        while(s[i] != ' ' && i >= 0){
            count++;
            i--;
            if(i<0) return count;
        }

        return count;
    }
};